archive/frames/views.py

```diff
@@ -139,8 +151,11 @@ def _get_aggregate_values(query_set, field, aggregate_field):
         else:
             return []
 
-    @list_route()
+    @action(detail=False)
```

Comment:
Updated due to DRF upgrade - see https://www.django-rest-framework.org/community/3.8-announcement/#action-decorator-replaces-list_route-and-detail_route